src/linagora.esn.videoconference.calendar/app/components/calendar-videoconference-form/calendar-videoconference.less

```
@@ -13,3 +13,40 @@
     flex-grow: 1;
   }
 }
+
+.conference-btns {
+  margin-left: 10px;
+  min-width: 0;
+}
+
+.btn-row {
+  display: flex;
+
+  a {
+    display: block;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+
+  i {
+    padding-right: 15px;
```

IMHO, the space between the icon and the text is too large. `4px` should be enough.

```suggestion
    padding-right: 4px;
```